### ISSUE TITLE
Video duplicated in playlist

### ISSUE THREAD

**Issue by regseb** [jaylinski/kodi-addon-vimeo#66]
### Describe the bug

When you add a video with JSON-RPC, the video is duplicated in the playlist.

### To Reproduce

1. In Kodi, _Allow remote control via HTTP_ (_Settings_ / _Services_ / _Control_).
2. Execute (by changing the IP address):
3. `curl -d '{ "jsonrpc": "2.0", "method": "Playlist.Clear", "params": { "playlistid": 1 }, "id": 1 }' -H 'Content-type: application/json' -X POST 127.0.0.1:8080/jsonrpc`
```JSON
{ "id": 1, "jsonrpc": "2.0", "result": "OK" }
```
4. `curl -d '{ "jsonrpc": "2.0", "method": "Playlist.Add", "params": { "playlistid": 1, "item": { "file": "plugin://plugin.video.vimeo/play/?video_id=306834650" } }, "id": 2 }' -H 'Content-type: application/json' -X POST 127.0.0.1:8080/jsonrpc`
```JSON
{ "id": 2, "jsonrpc": "2.0", "result": "OK" }
```
5. `curl -d '{ "jsonrpc": "2.0", "method": "Playlist.GetItems", "params": { "playlistid": 1, "properties": ["file"] }, "id": 3 }' -H 'Content-type: application/json' -X POST 127.0.0.1:8080/jsonrpc`
```JSON
{
    "id": 3,
    "jsonrpc": "2.0",
    "result": {
        "items": [{
            "file": "plugin://plugin.video.vimeo/play/?video_id=306834650",
            "label": "play",
            "type": "unknown"
        }],
        "limits": { "end": 1, "start": 0, "total": 1 }
    }
}
```
6. `curl -d '{ "jsonrpc": "2.0", "method": "Player.Open", "params": { "item": { "playlistid": 1 } }, "id": 4 }' -H 'Content-type: application/json' -X POST 127.0.0.1:8080/jsonrpc`
```JSON
{ "id": 4, "jsonrpc": "2.0", "result": "OK" }
```
7. `curl -d '{ "jsonrpc": "2.0", "method": "Playlist.GetItems", "params": { "playlistid": 1, "properties": ["file"] }, "id": 5 }' -H 'Content-type: application/json' -X POST 127.0.0.1:8080/jsonrpc`
```JSON
{
    "id": 5,
    "jsonrpc": "2.0",
    "result": {
        "items": [{
            "file": "plugin://plugin.video.vimeo/play/?video_id=306834650",
            "label": "play",
            "type": "unknown"
        }, {
            "file": "plugin://plugin.video.vimeo/play/?uri=%2Fvideos%2F306834650&texttracks=",
            "label": "Eddy De Pretto - Random",
            "type": "unknown"
        }],
        "limits": {"end": 2, "start": 0, "total": 2 }
    }
}
```

### Environment

- Kodi version: 19.4
- OSMC version: 2022.03-1
- Vimeo add-on version: 6.0.1

### Additional info

This is the first video that is played: `curl -d '{ "jsonrpc": "2.0", "method": "Player.GetProperties", "params": { "playerid": 1, "properties": ["position"] }, "id": 6 }' -H 'Content-type: application/json' -X POST 127.0.0.1:8080/jsonrpc`
```JSON
{ "id": 6, "jsonrpc": "2.0", "result": { "position": 0 } }
```

**Comment by jaylinski** [comment]
I added a fix in #70 if you want to test it.

**Comment by regseb** [comment]
I tested it in Kodi 20.5 and it works: I don't have duplicate video. 🚀

**Comment by jaylinski** [comment]
Thanks for the confirmation. I'll try to release the fix soonish.

**Comment by jaylinski** [comment]
Fix was released in v6.0.2.
